test/test_add_data.py

```diff
@@ -28,14 +28,13 @@ def test_add_all_data():
 
 
 def test_add_node_data():
-
     grid = two_triangles()
 
     # Create a UVar object for the depths:
     depths = UVar('depth', location='node', data=[1.0, 2.0, 3.0, 4.0])
     depths.attributes['units'] = 'm'
-    depths.attributes["standard_name"] = "sea_floor_depth"
-    depths.attributes["positive"] = "down"
+    depths.attributes['standard_name'] = 'sea_floor_depth'
```

Comment:
nope -- I don't care. I suppose we should be consistent, but I've actually never found it annoying or confusing to be arbitrary about it. I guess because my editor color-codes strings anyway.